src/SearchResults/index.js

```diff
@@ -79,13 +80,25 @@ function assignFacetStats(dest, facetStats, key) {
   }
 }
 
-function findMatchingHierarchicalFacetFromAttributeName(hierarchicalFacets, hierarchicalAttributeName) {
-  return find(
-    hierarchicalFacets,
-    function facetKeyMatchesAttribute(hierarchicalFacet) {
-      return includes(hierarchicalFacet.attributes, hierarchicalAttributeName);
-    }
-  );
+/**
+ * @typedef {Object} HierarchicalFacet
+ * @property {string} name
+ * @property {string[]} attributes
+ */
+
+/**
+ * @param {HierarchicalFacet[]} hierarchicalFacets
+ * @param {string} hierarchicalAttributeName
+ */
+function findMatchingHierarchicalFacetFromAttributeName(
+  hierarchicalFacets,
+  hierarchicalAttributeName
+) {
+  return find(hierarchicalFacets, function facetKeyMatchesAttribute(
+    hierarchicalFacet
+  ) {
+    return includes(hierarchicalFacet.attributes, hierarchicalAttributeName);
```

Comment:
This maybe can be replaced with 
```js
hierarchicalFacet.attributes.indexOf(hierarchicalAttributeName) > -1
```
?

Reply:
looks like lodash would also check `hierarchicalFacet.attributes !== null`
https://github.com/lodash/lodash/blob/4.17.11/lodash.js#L598

Reply:
Good call, I didn't replace that one since it's an array, but let's do it too